DESIGN.md

```diff
@@ -56,9 +56,9 @@ In most of the cases, the output of the copy backup is not suitable for direct c
 
 ## Partition group
 
-Partition group clusters log messages based on custom logic parametrized with message content. More precisely, a consumer in this group parses the message content to extract a list of partitions and places the message in a file with other messages that belong to those partitions. In the majority of cases, the message partition list contains a single value, the date corresponding to the timestamp recorded in the message.
+Partition group clusters log messages based on custom logic parametrized with message content. More precisely, a consumer in this group parses the message content to extract a list of partitions and places the message in a file with other messages that belong to those partitions. In the majority of cases, the message partition list contains a single value, the date corresponding to the mTimestamp recorded in the message.
```

Comment:
DESIGN.md and README.md were changed automatically when I ran package command on local. Will revert it in next commit.